src/test/mod.rs

```diff
@@ -973,11 +971,10 @@ fn rustfmt() -> PathBuf {
 #[test]
 fn verify_check_works() {
     let temp_file = make_temp_file("temp_check.rs");
-    assert_cli::Assert::command(&[
-        rustfmt().to_str().unwrap(),
-        "--check",
-        temp_file.path.to_str().unwrap(),
-    ])
-    .succeeds()
-    .unwrap();
+
+    Command::new(rustfmt().to_str().unwrap())
+        .arg("--check")
+        .arg(temp_file.path.to_str().unwrap())
+        .status()
+        .expect("check failed");
```

Comment:
nitpick: `check failed` feels generic, maybe a message like `run with check option failed` would be better.